src/components/TaskCommandLogs.tsx

```diff
@@ -132,7 +136,9 @@ class TaskCommandLogs extends React.Component<TaskCommandLogsProps> {
               </div>
             );
           }
-          return <TaskCommandRealTimeLogs initialLogLines={props.task.commandLogsTail || []} {...this.props} />;
+          return (
+            <TaskCommandRealTimeLogs initialLogLines={(props.task.commandLogsTail || []).concat()} {...this.props} />
```

Comment:
what's the empty .concat() for?

Reply:
To convert from `readonly strring[]` to just `string[]` 🤷‍♂

Reply:
😱